include/generators/lp_generator.h

```diff
@@ -0,0 +1,119 @@
+//
+// Created by panagiotis on 27/5/2019.
+//
+
+#ifndef VOLESTI_LP_GENERATOR_H
+#define VOLESTI_LP_GENERATOR_H
+
+#include "lp_problem.h"
+#include "polytope_generators.h"
+#include "polytopes.h"
+#include "Eigen"
+#include <cstdlib>
+
+template<class Point, typename NT>
+optimization::lp_problem<Point, NT> generate_lp_cube(int dim) {
+    typedef Eigen::Matrix<NT, Eigen::Dynamic, 1> VT;
+    VT obj;
+    obj.resize(dim);
+
+    for (int i = 0; i < dim; i++) {
+        obj(i) = -5 + ((double)rand() / RAND_MAX)*30;
+    }
+
+    HPolytope<Point> HP = gen_cube<HPolytope<Point> >(dim, false);
+
+    return optimization::lp_problem<Point, NT>(HP, obj, optimization::minimize);
+}
+
+template<class Point, typename NT>
+optimization::lp_problem<Point, NT> generate_lp_cross(int dim) {
+    typedef Eigen::Matrix<NT, Eigen::Dynamic, 1> VT;
```

Comment:
actually all those function contain the same (or nearly the same) code. Please create some help function to avoid repeating code.